src/config/gameEffects/gameEffects.spec.ts

```diff
@@ -17,7 +17,7 @@ jest.mock('../rounds', () => {
             cost: 2,
           },
         },
-        effect: () => ({ capacity: 10 }),
+        effect: () => ({ capacityChange: 10 }),
```

Comment:
the return value of our mocks is currently not strongly typed therefore the vscode refactor missed this one